src/lib/eina/eina_sched.c

```diff
@@ -17,68 +17,22 @@
  */
 
 #ifdef HAVE_CONFIG_H
-# include "config.h"
+#include "config.h"
 #endif
+#include "eina_thread.h"
 
-#include <pthread.h>
 #ifdef __linux__
-# include <sched.h>
-# include <sys/time.h>
-# include <sys/resource.h>
-# include <errno.h>
+#include <sched.h>
+#include <sys/time.h>
+#include <sys/resource.h>
+#include <errno.h>
```

Comment:
We don't need those on Win32?

Reply:
yep, i'll create a `_sched_{win32,posix}.c` and put there